src/marqo/core/structured_vespa_index/structured_vespa_index.py

```diff
@@ -52,6 +52,78 @@ class StructuredVespaIndex(VespaIndex):
     def __init__(self, marqo_index: StructuredMarqoIndex):
         self._marqo_index = marqo_index
 
+    def to_vespa_partial_document(self, marqo_document: Dict[str,Any]) -> Dict[str, Any]:
+        vespa_id: Optional[str] = None
+        vespa_fields: Dict[str, Any] = dict()
+        score_modifiers: Dict[str, float] = dict()
+
+        if constants.MARQO_DOC_ID not in marqo_document:
+            raise MarqoDocumentParsingError(f"Marqo encountered an error when parsing the partial update document "
```

Comment:
I think the message is too verbose. Just " '{constants.MARQO_DOC_ID}' is a required field "
                                            f"but it does not exist." should suffice. Also printing our the whole dict seems like a bad idea